python/tests/test_valid_topic_data.py

```diff
@@ -163,3 +163,41 @@ def test_retention() -> None:
             error = f"Invalid retention for topic {topic_name}: {retention}"
 
         assert retention in allowed, error
+
+
+def test_dlq_configuration() -> None:
+    # These topics do not match the naming conventions
+    custom_dlq_mapping = {
+        "ingest-generic-metrics-dlq": "ingest-performance-metrics",
+        "snuba-dead-letter-replays": "ingest-replay-events",
+        "snuba-dead-letter-metrics": "snuba-metrics",
+        "snuba-dead-letter-querylog": "snuba-queries",
+        "snuba-dead-letter-generic-metrics": "snuba-generic-metrics",
+    }
+
+    topics_dir = _TOPICS
+    for filename in topics_dir.iterdir():
+        if filename.stem.endswith("-dlq"):
+            main_topic_name = custom_dlq_mapping.get(filename.stem, filename.stem[:-4])
```

Comment:
What does -4 represent here? Can you define a constant for it and use it instead?

Similar comment for this line as well https://github.com/getsentry/sentry-kafka-schemas/pull/319/files#diff-5977ee087c598ee3b0e4480050abfd5d9c622ab72ea34ebf9cbdaa364b397d11R184.

Reply:
updated